client/my-sites/google-my-business/controller/index.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * External Dependencies
+ *
+ * @format
+ */
+
+import React from 'react';
+
+/**
+ * Internal Dependencies
+ */
+
+import GMBSelectBusinessType from '../select-business-type';
+
+export function show( context, next ) {
```

Comment:
We're going to add many pages in the next iterations. What about naming this with a more explicit name such as `selectBusinessType`?